src/browser/api/bitcoind/__tests__/bitcoindapi.test.ts

```diff
@@ -0,0 +1,30 @@
+import { getBaseUrl } from '../index'
+
+describe('bitcoind api tests', () => {
```

Comment:
a name related to `getBaseUrl` function would be more appropriate.
Same with the file

Reply:
renamed both